script/c511005067.lua

```diff
@@ -0,0 +1,54 @@
+--Sealer Formation
+--  By Shad3/ fixed for errors by GameMaster
+
+local scard=c511005067
+
+function scard.initial_effect(c)
+  --Activate
+  local e1=Effect.CreateEffect(c)
+  e1:SetType(EFFECT_TYPE_ACTIVATE)
+  e1:SetCode(EVENT_FREE_CHAIN)
+  c:RegisterEffect(e1)
+  --Add/Draw
+  local e2=Effect.CreateEffect(c)
+  e2:SetType(EFFECT_TYPE_FIELD+EFFECT_TYPE_TRIGGER_O)
+  e2:SetCode(EVENT_PREDRAW)
+  e2:SetRange(LOCATION_SZONE)
+  e2:SetCondition(scard.cd)
+  e2:SetOperation(scard.op)
+  c:RegisterEffect(e2)
+  --Global const
+  if not scard.gl_chk then
+    scard.gl_chk=true
+    scard.seal_lst={
+      25880422,29549364,63102017,
+      58921041,511000176
+    }
+  end
+end
+
+function scard.cd(e,tp,eg,ep,ev,re,r,rp)
+  return tp==Duel.GetTurnPlayer() and Duel.GetFieldGroupCount(tp,LOCATION_DECK,0)>0 and Duel.GetDrawCount(tp)>0 and Duel.IsExistingMatchingCard(scard.fil,tp,LOCATION_GRAVE,0,1,nil)
+end
+
+function scard.fil(c)
+  for i,i in pairs(scard.seal_lst) do

```

Comment:
for i,i in pairs(scard.seal_lst) do
i would sugest to use diferent variables for key and value like
for a,i in pairs(scard.seal_lst) do
>.<